### PR TITLE
Add Gnosis support

### PR DESCRIPTION
**Pull request by unxnn**

### CODE REVIEW

**Comment by tsudmi** [comment]
@unxnn I think it's better to rename to gnosis as it's used in their deposit CLI: https://github.com/gnosischain/validator-data-generator

**Comment by unxnn** [comment]
> @unxnn I think it's better to rename to gnosis as it's used in their deposit CLI: https://github.com/gnosischain/validator-data-generator

@tsudmi renamed

File: charts/lighthouse/templates/statefulset.yaml [hunk 1]
```diff
@@ -47,16 +47,25 @@ spec:
...
             - lighthouse
             - bn
             - --staking
             - --http-address=0.0.0.0
             - --eth1-endpoints={{ .Values.eth1Endpoints | join "," }}
             - --datadir=/data
+          {{- if ne .Values.network "gnosis" }}
             - --network={{ .Values.network }}
+          {{- else }}
+            - --testnet-dir=/data/gnosis-config
```
[the remark this answers is not in the document]
Comment (unxnn) [reply charts/lighthouse/templates/statefulset.yaml:67]: testnet is for gnosis, `if ne` = not equal @tsudmi

File: charts/operator/values.yaml [hunk 2]
```diff
@@ -67,9 +68,15 @@ image:
...
+  prysmGnosis:
+    repository: "ghcr.io/gnosischain/gbc-prysm-validator"
+    tag: "v2.0.4-gbc"
   lighthouse:
     repository: "sigp/lighthouse"
     tag: "v2.0.1"
+  lighthouseGnosis:
+    repository: "ghcr.io/gnosischain/gbc-lighthouse"
+    tag: "v2.0.1-gbc"
   teku:
```
[the remark this answers is not in the document]
Comment (unxnn) [reply charts/operator/values.yaml:80]: @tsudmi added: https://github.com/stakewise/helm-charts/pull/82/commits/9e3660d6e828cb59795d6468d9a0c7eb433e72b2

**Pull request merged**